pkgs/development/python-modules/ifcfg/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, fetchPypi
+, buildPythonApplication
+, python
+, iproute2
+}:
+
+buildPythonApplication rec{
```

```suggestion
buildPythonPackage rec {
```
This is primarily a library.